certify-service/src/test/resources/schema.sql

```diff
@@ -0,0 +1,111 @@
+CREATE TABLE IF NOT EXISTS client_detail(
```

Comment:
We also need a similar db script folder for the devops to deploy in the env
https://github.com/mosip/esignet/tree/develop/db_scripts and https://github.com/mosip/esignet/tree/develop/db_upgrade_script

Reply:
added relevant scripts for 3 tables used in keymanager

Reply:
db_upgrade changes are not available

Reply:
added db_upgrade folder

Reply:
below files can be removed since it is the first time db is getting introduced.

https://github.com/mosip/inji-certify/tree/736d8c68b25caa0f962b712714946ec93a25ad63/db_upgrade_script/mosip_certify/sql